modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/Instrument.scala

```diff
@@ -9,7 +9,7 @@ import scalaz.concurrent.Task
 trait Instrument extends System {
   // The name used for this instrument in the science fold configuration
   val sfName: String
-  def observe(config: Config): SeqObserve[DhsClient, ObserveResult]
+  def observe(config: Config): SeqObserve[(DhsClient, List[Header]), ObserveResult]
```

Comment:
I guess we need to link this to the client level request to, to send client headers